### ISSUE TITLE
Vue.js is detected but the Vue Panel is not showing

### ISSUE THREAD

**Issue by rafito** [vuejs/vue-devtools#1209]
### Version
5.3.3

### Browser and OS info
Chrome 83.0.4103.97 / Windows 10

### Steps to reproduce
After enter the page Vue.js is detected on this page. The message "Open DevTools and look for the Vue panel" at the Vue-devtools icon.

### What is expected?
Appears Vue Panel on DevTools

### What is actually happening?
It's not appearing

**Comment by landocalris** [comment]
I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around. 

1. Go to the DevTools settings
2. Under Apperance, choose a different theme
3. The Vue Panel should be back now in DevTools
4. Go back to dev tools settings and put your theme back to what you had before
5. The Vue Panel should still be there

I hope this works for you too.

**Comment by Fax** [comment]
I have the same issue here with Chrome Version 83.0.4103.97 (Official Build) (64-bit).
Updating the theme didn't fix the issue, the vue tab is still not visible.

**Comment by diazemiliano** [comment]
F#@k!!!, I've lost many many time with this!!!! Thanks @landocalris

**Comment by CobyYates** [comment]
Worked for me! Thanks @landocalris

**Comment by ChrisBFusion** [comment]
The above workaround does not work in Firefox.

**Comment by rustinlewis** [comment]
I'm having the same problem and none of these suggestions nor any other I've been able to google is causing it to show again.  Dang it's much harder to developer when I can't see my state.

macOs Catalina 10.15.5 
Version 83.0.4103.106 (Official Build) (64-bit)

**Comment by pedro380085** [comment]
Here doesn't work either.

**Comment by clemsontiger** [comment]
Well this doesn't seem to always fix the issue for me. very frustrating

**Comment by themnd** [comment]
I can confirm the workaround (https://github.com/vuejs/vue-devtools/issues/1209#issuecomment-641916382) with Chrome Version 84.0.4147.105 (Official Build) (64-bit) on mac os x 10.14.6

**Comment by TheKeymaster** [comment]
Make sure that Vue is not accidentally set in production mode. For me changing it to Development Mode fixed the issue.

![image](https://user-images.githubusercontent.com/20555112/89774065-6ab83d00-db05-11ea-8425-7a0a63427bfd.png)

**Comment by yuriybabenko** [comment]
Same problem on OSX 10.15.6 and Chrome 85.0.4183.83. None of the above-mentioned solutions have worked.

**Comment by HmHasan** [comment]
> I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around.
> 
> 1. Go to the DevTools settings
> 2. Under Apperance, choose a different theme
> 3. The Vue Panel should be back now in DevTools
> 4. Go back to dev tools settings and put your theme back to what you had before
> 5. The Vue Panel should still be there
> 
> I hope this works for you too.

**Comment by wellssu0** [comment]
1. Switch on or off the plugin several times, and then change the theme    
2. Or turn off the devtools of vue2, and then change the theme

**Comment by HmHasan** [comment]
Thank You Its Work For Me

**Comment by ahprasandh** [comment]
This worked for me.

https://github.com/vuejs/vue-devtools/issues/620#issuecomment-699594776

**Comment by martin-masterclass** [comment]
Works like a charme! Thank you! Here a screenshot of how to find the appearance settings for dummies like me :-D
![image](https://user-images.githubusercontent.com/64164118/94556322-2cb1dc80-025d-11eb-8e57-ef899eca99fd.png)

**Comment by tommydemarco** [comment]
> 1. Switch on or off the plugin several times, and then change the theme
> 2. Or turn off the devtools of vue2, and then change the theme

This worked for me. Thanks.

**Comment by MinSomai** [comment]
turning off the vue2 devtools and changing the theme worked for me.

**Comment by kevinsbath** [comment]
Nothing worked for me - until I opened the devtools settings, and clicked 'restore defaults and reload" in the bottom right of the settings screen.

Vue pane now visible. :-)

**Comment by blizzardKv** [comment]
work for me - up vue-router to vue-router@next, firstly just installed router for vue2

**Comment by ATJohnson83** [comment]
Deleting and re-installing the vue devtools (beta version) plus restoring the devtools defaults for chrome did the trick for me.

**Comment by giantthinker** [comment]
> I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around.
> 
> 1. Go to the DevTools settings
> 2. Under Apperance, choose a different theme
> 3. The Vue Panel should be back now in DevTools
> 4. Go back to dev tools settings and put your theme back to what you had before
> 5. The Vue Panel should still be there
> 
> I hope this works for you too.

Thank you so. It works.

**Comment by SlimDeluxe** [comment]
Tried all the suggestions and none worked for me with the beta plugin and Vue 3.
On browser Chromium, Chrome and Brave, on Ubuntu 20.04.

**Comment by hsyntnc** [comment]
I just removed the regular vue devtools and installed beta version for vue3 and it worked for me.

**Comment by jovialcore** [comment]
> I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around.
> 
> 1. Go to the DevTools settings
> 2. Under Apperance, choose a different theme
> 3. The Vue Panel should be back now in DevTools
> 4. Go back to dev tools settings and put your theme back to what you had before
> 5. The Vue Panel should still be there
> 
> I hope this works for you too.

Your solution worked for me. Thanks

**Comment by samuel012** [comment]
> I just removed the regular vue devtools, installed beta version for vue3 and it worked for me.

Actually, this worked for me, but I had to do an additional step. All in all, these are the steps:

1. remove regular vue devtools
2. remove the **new** vue devtools (since i have them both installed)
3. re-add the **new** vue devtools

That's it.

**Comment by yxw007** [comment]
> > I just removed the regular vue devtools, installed beta version for vue3 and it worked for me.
> 
> Actually, this worked for me, but I had to do an additional step. All in all, these are the steps:
> 
> 1. remove regular vue devtools
> 2. remove the **new** vue devtools (since i have them both installed)
> 3. re-add the **new** vue devtools
> 
> That's it.

thanks ~  haha

**Comment by B45i** [comment]
> I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around.
> 
> 1. Go to the DevTools settings
> 2. Under Apperance, choose a different theme
> 3. The Vue Panel should be back now in DevTools
> 4. Go back to dev tools settings and put your theme back to what you had before
> 5. The Vue Panel should still be there
> 
> I hope this works for you too.

Install vue dev tool beta from [here](https://chrome.google.com/webstore/detail/vuejs-devtools/ljjemllljcmogpfapbkkighbhhppjdbg?hl=en), then try these steps

**Comment by ogelin** [comment]
None of these work for me, still facing the issue.

**Comment by edcast45** [comment]
Thanks, @B45i this helped.  Also, it may help to have the Beta version instead of the regular version.

**Comment by camohub** [comment]
Beta version works. But why stable version not?

**Comment by Akryum** [comment]
@camohub Stable doesn't support Vue 3

**Comment by Zonciu** [comment]
This works for 6.x version, you need to remove or disable 5.x version before doing this.

> I found [this ](https://github.com/vuejs/vue-devtools/issues/1072#issue-497741297) suggestion that worked for me. It is a good work around.
> 
> 1. Go to the DevTools settings
> 2. Under Apperance, choose a different theme
> 3. The Vue Panel should be back now in DevTools
> 4. Go back to dev tools settings and put your theme back to what you had before
> 5. The Vue Panel should still be there
> 
> I hope this works for you too.

**Comment by J-Rojas** [comment]
The workaround does't work for me on Chromium v83.

**Comment by kalinchernev** [comment]
Yep, turning the extension off and on again actually did work 
https://www.youtube.com/watch?v=p85xwZ_OLX0

**Comment by EasonApolo** [comment]
My case is I used a internal vue.js, which does not have a .min extension, but its content seems like a minimized version... If those workarounds not help, maybe you should check your vue resource is not vue.min.js

**Comment by esmaydogdu** [comment]
I installed the beta version, and I also had to play with the settings, finally worked! Thank you!

**Comment by Qzhor** [comment]
remove all the plugins, just like react, redux, vue-performance, old vue devtool....
restart your chrome
wow ,it works!

**Comment by kamil-kielczewski** [comment]
For me this was critical:

When I install plugin for short moment in upper right chrome corner the VUE ICON appears with message (screenshot is not my):

![image](https://user-images.githubusercontent.com/15379981/131952968-003247b0-a219-472a-8393-8158b763038f.png)

And CRITICAL is to click right button on that icon and choose "PIN" (it to extensions icons bar) - if you not do it - then you will never see that icon again (which is mistake). After that shut down and reopen chrome. And now in you page with vue, this icon sould have  green colors (you can click on it) - and in chrome console you should see

![image](https://user-images.githubusercontent.com/15379981/131953367-961576ac-3a78-4611-a3cc-8d0b5d7f721f.png)

**Comment by jaredmeadowsLB** [comment]
I had already installed the Beta version, removed the old version of Vue devtools, but it still wasn't working, then I finally closed devtools and reopened them, and the tab showed up!
TL;DR: if nothing works for you, try just closing the panel and reopen the Chrome devtools panel!

**Comment by Akryum** [comment]
https://devtools.vuejs.org/guide/faq.html#the-vue-devtools-don-t-show-up

**Comment by bashunaimiroy** [comment]
For what it's worth, reloading the Chrome Devtools window also works, and I suspect the "new theme" trick works because it includes a Chrome Devtools reload under the hood. 

The MacOS keyboard (when Devtools is focused, at least) is `Option+R`. You can also bring up the Command Palette in Devtools and type "reload". 

Note: This is incredibly and inexplicably hard to Google, and it's not documented in Devtools docs as far as I can tell. This is really bizarre, because it seems like pretty core functionality and is very helpful in cases like these.

![image](https://user-images.githubusercontent.com/27756206/150000200-6aecda1e-1df9-411c-8215-77360fd3f526.png)

**Comment by imsimonpham** [comment]
I tried closing the dev tool and did a hard refresh (shift + click the reload icon on Mac) and it worked.

**Comment by clwilliams8** [comment]
> 

How do you open "Devtool Settings" ? I don't see this?

**Comment by bashunaimiroy** [comment]
> > 
> 
> How do you open "Devtool Settings" ? I don't see this?

@clwilliams8 it's the gear icon. But my reload trick should work and is quicker, let me know if it's working for you?

**Comment by clwilliams8** [comment]
thanks for the help. for me it ended up that i wasn't running my vite complier 🤷🏼‍♂️

**Comment by annajar222** [comment]
I tried everything and it just did not work for me at all. and btw how do I install the beta version !

**Comment by DeWetSwipedOn** [comment]
Restore defaults and reload in Dev Tools settings worked for me

**Comment by vinit095** [comment]
In my case I just restored to default settings and **viola** it's showin in panel now.

![image](https://github.com/vuejs/devtools/assets/22002287/e9159743-6e9f-4a5c-9313-b012cdba0af2)

**Comment by hecktarzuli** [comment]
@vinit095 's solution is the only thing that worked for me.  I tried un/re-installing the extension, upgraded Chrome to `122.0.6261.94`, rebooted my Mac, nothing worked except this.

**Comment by valterbarros** [comment]
> Restore defaults and reload in Dev Tools settings worked for me

Oh thanks that solved my problem!

**Comment by markelzhang** [comment]
I try all methods mentioned above but not work yet. After reset devtool setting, it appears:

1. Go to the DevTools settings
2. Click `restore defaults and reload`

<img width="934" alt="image" src="https://github.com/vuejs/devtools/assets/61811312/c414bbd3-aad7-46f1-9243-9323e5647ddd">

**Comment by juanbomfim22** [comment]
None of the above solutions works when using **Edge Version 124.0.2478.97 (Official build) (64-bit)**

**Comment by julesgilson** [comment]
It's not a great look on Vue that this issue is still unresolved after 4 years - I am assuming this project is dead, so basically no devtools  for older versions of Vue. Working on the new and sparkly is all good and well, but one day it will be old and forgotten but some poor soul is still going to have to support it / migrate it.

As far as I can tell, some or all of the work-arounds noted by others work - temporarily.

I guess the message is - don't use Vue if you expect your tools to work in a few years.

**Comment by riceball1** [comment]
2025, just had this issue on chrome, I removed it entirely since the setting was already on for incognito and it works again. Happened on `Version 134.0.6998.166 (Official Build) (arm64)`

**Comment by defghy** [comment]
If you still use Vue@2, try our fork with performance optimizations from version 5.3.4.

https://github.com/kxxxlfe/devtools/releases
